### PR TITLE
Batch fulfill orders + update lastCheckedBlock

### PR DESCRIPTION
**Pull request by Velenir**
As suggested by @anxolin _one action to rule them all_ for fulfilling orders + updating `lastCheckedBlock`

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
   * **🎩 [Dapp](https://pr69--gpv2swap.review.gnosisdev.com)**: Testing web app

**Review by W3stside — APPROVED** [review]
very nice!

File: src/custom/state/orders/updater.ts [hunk 1]
```diff
@@ -150,47 +151,45 @@ export function EventUpdater(): null {
...
-      // only update lastCheckedBlock if no logs
-      if (logs.length === 0) {
-        // SET lastCheckedBlock = lastBlockNumber
-        dispatch(updateLastCheckedBlock({ chainId, lastCheckedBlock: lastBlockNumber }))
-        return
-      }
-
       const block2DateMap = await buildBlock2DateMap(library, logs)
 
+      const ordersBatchData: (OrderFulfillmentData & Pick<Log, 'transactionHash'>)[] = logs.map(log => {
```
Comment (anxolin) [inline src/custom/state/orders/updater.ts:156]: i would avoid complex inline ts expressions
the line has already a declaration and a map, adding this makes it harder to read

File: src/custom/state/orders/hooks.ts [hunk 2]
```diff
@@ -124,6 +127,14 @@ export const useFulfillOrder = (): FulfillOrderCallback => {
   return useCallback((fulfillOrderParams: FulfillOrderParams) => dispatch(fulfillOrder(fulfillOrderParams)), [dispatch])
 }
 
+export const useFulfillOrdersBatch = (): FulfillOrdersBatchCallback => {
```
Comment (anxolin) [inline src/custom/state/orders/hooks.ts:130]: I was reading `useful fill orders` 😂

**Review by anxolin — COMMENTED** [review]
Great!

**Pull request merged**